src/JsonConfig.cs

```diff
@@ -65,8 +65,16 @@ public static void LoadConfig()
 
             if (!firstRun)
             {
-                string jsonText = File.ReadAllText("settings.conf");
-                settings = JsonConvert.DeserializeObject<AppConfig>(jsonText);
+                try
+                {
+                    string jsonText = File.ReadAllText("settings.conf");
+                    settings = JsonConvert.DeserializeObject<AppConfig>(jsonText);
+                }
+                catch (JsonReaderException)
+                {
+                    MessageBox.Show("Your configuration file was corrupt and has been reset to the default settings.");

```

Comment:
Please change to the following to give the MessageBox a title and icon, and identify the app it is associated with:
`System.Windows.Forms.MessageBox.Show("Your WinDynamicDesktop configuration file was corrupt and has been reset to the default settings.", "Warning", MessageBoxButtons.OK, MessageBoxIcon.Warning);`

Reply:
Agreed. Will get to it later. Shouldn't we add this information this MessageBox style everywhere though? It seems there are some MessageBoxes around that don't have any icon or app reference.

Reply:
You're probably right, the message boxes should have more consistent styling. I should create a static helper class for showing them that could handle some of this.

Reply:
For now I've just fixed this one. Doing something like that would be great for consistency.